modules/building/pages/prefetching-dependencies.adoc

```diff
@@ -172,35 +180,65 @@ requests==2.32.2 \
 
+=== [[pip-walkthrough]]`pip` walkthrough
+
+Refer to the link:https://hermetoproject.github.io/hermeto/pip/#example[pip walkthrough] for a detailed example of using pip with Hermeto.
+
+IMPORTANT: You *must* take into account the differences between using Hermeto in a pipeline, and using `hermeto` as a command-line tool.
+
 == [[npm]]Enabling prefetch builds for `npm`
-Cachi2 supports `npm` by fetching any dependencies you declare in your `package.json` and `package-lock.json` project files. The npm CLI manages the `package-lock.json` file automatically, and Cachi2 fetches any dependencies and enables your build to install them without network access.
+Hermeto supports `npm` by fetching any dependencies you declare in your `package.json` and `package-lock.json` project files. The npm CLI manages the `package-lock.json` file automatically, and Hermeto fetches any dependencies and enables your build to install them without network access.
 
 .Prerequisites
 * You have an up-to-date link:https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json[`package-lock.json`] file, newer than version 1, in your source repository. To make sure that you have the latest `package-lock.json` file, or to create a lockfile, run the link:https://docs.npmjs.com/cli/v9/commands/npm-install?v=true[`npm-install`] command. You can also look at the `lockfileVersion` attribute in your `package-lock.json` file to make sure its value is a number greater than `*1*`.
 
+=== [[npm-walkthrough]]`npm` walkthrough
+
+Refer to the link:https://hermetoproject.github.io/hermeto/npm/#example[npm walkthrough] for a detailed example of using npm with Hermeto.
+
+IMPORTANT: You *must* take into account the differences between using Hermeto in a pipeline, and using `hermeto` as a command-line tool.
+
 == [[yarn]]Enabling prefetch builds for `yarn`
 
-Supported versions: 1.x and 3.x. Cachi2 automatically detects the version of `yarn` and fetches any dependencies you declare in your `package.json` and `yarn.lock` project files.
+Supported versions: 1.x and 3.x. Hermeto automatically detects the version of `yarn` and fetches any dependencies you declare in your `package.json` and `yarn.lock` project files.
 
 .Prerequisites
 
-* You have an up-to-date `yarn.lock` file in your source repository. To ensure you have the latest `yarn.lock` file or to create it, run the `yarn install` command. If `yarn.lock` is not up-to-date, Cachi2 will not fetch the dependencies.
+* You have an up-to-date `yarn.lock` file in your source repository. To ensure you have the latest `yarn.lock` file or to create it, run the `yarn install` command. If `yarn.lock` is not up-to-date, Hermeto will not fetch the dependencies.
+
+=== [[yarn-walkthrough]]`yarn` walkthrough
+
+Refer to the link:https://hermetoproject.github.io/hermeto/yarn/#example[yarn walkthrough] for a detailed example of using yarn with Hermeto.
```

Comment:
It might be worth also referring to the Yarn 1.x example since they operate differently

Reply:
I got some push back on this in the Hermeto docs refresh. Let me check that PR...